gazebo_ros/scripts/debug

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 final="$@"
 
+# remove ros remapping arguments like "__name:=node_name"; otherwise, they'd 
+# be interpreted as a world name by gzserver, and it'd crash while loading it
+final=`echo "$final" | sed 's/[^ ]*:=[^ ]* \?//g'`
```

Comment:
we probably don't need the echo call, does `sed 's/[^ ]*:=[^ ]* \?//g' <<< $final` works for you?